tests/unit/lib/util/LocationStatusManager.spec.js

```diff
@@ -0,0 +1,366 @@
+'use strict'; // eslint-disable-line
+
+const sinon = require('sinon');
+const async = require('async');
+const assert = require('assert');
+
+const fakeLogger = require('../../../utils/fakeLogger');
+const locationConfig = require('../../../../conf/locationConfig.json') || {};
+const LocationStatus = require('../../../../lib/models/LocationStatus');
+const LocationStatusManager = require('../../../../lib/util/LocationStatusManager');
+
+const zkNodes = {
+    '/backbeat/crr/state/us-east-1': {
+        paused: false,
+    },
+    '/backbeat/crr/state/us-east-2': {
+        paused: false,
+    },
+};
+
+const fakeMongoClient = {};
+const fakeZkClient = {
+    getData: (path, cb) => cb(null, Buffer.from(JSON.stringify(zkNodes[path]))),
+};
+const fakeRedisClient = {};
+
+describe('LocationStatusManager', () => {
+    let lsm;
+
+    beforeEach(() => {
+        lsm = new LocationStatusManager(
+            ['lifecycle'],
+            fakeMongoClient,
+            fakeZkClient,
+            fakeRedisClient,
+            {
+                crr: 'crr-topic',
+                ingestion: 'ingestion-topic',
+            },
+            {
+                crr: {
+                    namespace: '/backbeat/crr',
+                    statePath: '/state',
+                },
+                ingestion: {
+                    namespace: '/backbeat/ingestion',
+                    statePath: '/state',
+                },
+            },
+            fakeLogger,
+        );
+    });
+
+    afterEach(() => {
+        sinon.restore();
+    });
+
+    it('pauseService:: should pause selected service', done => {
```

Comment:
nit: `pauseService::` should be in a describe block instead, to keep the "flow" of describing the tests